src/components/Notification.tsx

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+
+/* eslint-disable jsx-a11y/control-has-associated-label */
+interface NotificationProps {
+  error: string,
+}
+
+export const Notification = ({ error }:NotificationProps) => {
+  const [isHidden, setIsHidden] = useState(false);
+
+  return (
+    <div className={`notification is-danger is-light has-text-weight-normal ${isHidden ? 'hidden' : ''}`}>
```

Comment:
it's better to move condition to separate variable and also use correct code style with line breaks